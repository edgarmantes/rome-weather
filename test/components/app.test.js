import React from 'react'
import App from '../../src/components/app';
import renderer from 'react-test-renderer';
import Tabs from '../../src/components/tabs';

describe('<App />' , () => {


	let wrapper

	beforeEach( ()=>{
		wrapper = shallow(<App  />)
	})

	it('Renders App component properly', () => {
		expect(wrapper.length).toEqual(1)
	});

	it('Snapshot of App', () => {
        const renderedValue =  renderer.create(<App />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });
});

