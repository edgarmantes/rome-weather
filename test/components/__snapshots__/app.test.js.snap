// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Snapshot of App 1`] = `
<div
  className="app-container"
>
  <div
    className="header-container"
  >
    <h1>
      Rome's Weather
    </h1>
  </div>
</div>
`;
